cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -82,3 +82,249 @@ def test_provider_summary_topology():
+            3. The "Compare Host / Node" view should be displayed.
+            - icons are displayed for all and only selected hosts with hostname displayed
+            - one of the host icons has the host denoted as "(base)" in the hostname
+            - host properties row is displayed (default)
+            - "% Matched" text or graphs are displayed
+            - when 3 or more hosts are displayed, remove icons exist for all non-base hosts.
+            4. "Configuration" in comparison sections should turn blue.
+            5. New rows should be displayed in the compare host/node view for all configuration
+            items (OS, Guest Applications, ...). The new rows should have the following:
+            - "% Matched" text or graphs are displayed
+            - sub-items that are not empty should have the ">" displayed and be able to be
+            expanded.
+            6. The configuration row/section should be expanded, displaying all sub-items in the
+            section.
+            - Items that do not match the base host should be in blue.
+            - The number of properties displayed should match the number denoted on the parent row.
+            - Properties for non-base hosts should be in purple/dark blue.
+            7. The section should collapse again to only display the parent row.
+            8. Same expected results as for step 6 and 7.
+    '''
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.infra_hosts
+def test_host_comparison_my_company_tags():
+    '''
+    Verify host comparisons view functionality for 'My Company Tags' section.
+    This is going to be the same test as test_host_comparison_properties, but for the "My Company
+     Tags" section.
+
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select at least 2 hosts (three if possible) by checking the box in upper left of
+            host icons.
+            3. Click "Compare Selected Items" under the "Configuration" dropdown.
+            4. Click to select/check the "My Company Tags" checkbox in "Comparison Sections".
+            5. Click "Apply"
+            *If the displayed rows do not have any items (item count is '(0)') the next steps do
+            not apply.
+            6. If the "My Company Tags" rows/sections have sub-items, click on one of the "My
+            Company Tags" row items (Department, Customer, ...).
+            7. Click on the same "My Company Tags" item again.
+            8. Repeat steps 6 and 7 for all "My Company Tags" sub-items.
+            *Steps 6 and 7 apply recursively to any sub items of parent sections that may exist.
+        expectedResults:
+            1. Hosts view is displayed with hosts filtered via the Default filter (denoted by
```

Comment:
This expectedResult could be left out (just left as `1. `) or could be shorted to be `view is displayed`. Similarly for the other expectedResults that are just for navigation.